src/test/java/uk/gov/cshr/repository/InviteRepositoryTest.java

```diff
@@ -83,6 +84,20 @@ public void existsByCodeAndStatusReturnsCorrectResult() {
         assertThat(existsByCodeAndStatusForExpiredInvite, equalTo(false));
     }
 
+    @Test
+    public void findByForEmailAndStatusShouldReturnCorrectInvite() {
+        Invite invite = createInvite();
```

Comment:
Please call createInvite("123abc", "test@example.org") here instead on calling createInvite() which will increase the readability of the code.

Reply:
Two new constant defined for the above string values and used in the class as suggested.